node/modules/lp2p/rcmgr.go

```diff
@@ -179,6 +180,7 @@ type rcmgrMetrics struct{}
 
 func (r rcmgrMetrics) AllowConn(dir network.Direction, usefd bool) {
 	ctx := context.Background()
+	ctx, _ = tag.New(ctx, tag.Upsert(metrics.Network, buildconstants.NetworkBundle))
```

Comment:
see if we can aim for this diff to have 1 instance of `buildconstants.NetworkBundle`

```suggestion
	ctx = metrics.AddNetworkTag(ctx)
```